### DISCUSSION TITLE
Custom Setting: Rollup Settings

### DISCUSSION THREAD

**Discussion by nhardenVenn** [jamessimone/apex-rollup#335] [Q&A] [answered]
Hello!

I just installed in sandbox, and I'm starting on the setup. The first step is to enable the Default Organization Level Value under the Rollup Settings. However, I have no record to edit here.

<img width="1919" alt="Screen Shot 2022-06-21 at 9 58 47 AM" src="https://user-images.githubusercontent.com/107888703/174831942-e87329fc-6695-4f51-a410-a3ca5b45bd66.png">

Have I already missed something?

Thank you.

**Comment by nhardenVenn** [comment, marked as answer]
Never mind. I see more instruction in the Read Me.

**Reply by jamessimone** [reply]
No worries! Glad you figured it out. Let me know if other questions come up.
